Fix typo in ReadableStreamBYOBReader.read() code example (#21267)
mdn/content: files/en-us/web/api/readablestreambyobreader/read/index.md

      .then(function processBytes({ done, value }) {
        // Result objects contain two properties:
        // done  - true if the stream has already given all its data.

>         // value - some data. Always defined, even when done is true.

Reviewer: ```suggestion
        // value - some data. 'undefined' if the reader is canceled.
```
